### ISSUE TITLE
Toggle Natural Scrolling script does not work on Sonoma

### ISSUE THREAD

**Issue by benediktms** [raycast/script-commands#886]
The script fails to execute. I suspect it is because the system preferences pane changes based on whether the OS detects if there is a mouse connected, if there is, 'Trackpad' becomes 'Mouse' and displays different settings. Not sure how this behaves with Apple's Magic Mouse or Magic Trackpad.

**Comment by dehesa** [comment]
Yes, it look like the screen has indeed change. Now you need to go to the "Scroll & Zoom" section and then click the toggle named "Natural scrolling". @wilmarques are you able to perform this changes?
![NaturalScrolling](https://github.com/raycast/script-commands/assets/684318/d849f10b-4227-4a53-9549-f46592fdbdb7)

**Comment by wilmarques** [comment]
Yes! Maybe not on the following days, but I'll fix it! :)

**Comment by dehesa** [comment]
Hey @wilmarques, please don't forget about this.

![pc-help-gif](https://github.com/raycast/script-commands/assets/684318/1397c530-5aa4-4e98-a47a-02dcea80f45d)

**Comment by dehesa** [comment]
I will be closing this for now, but I am happy to re-open it whenever there is a bit more activity.
